pages/product.js

```diff
@@ -2,9 +2,25 @@ const { I } = inject();
 
 module.exports = {
   productPrice: { css: '#our_price_display' },
+  addToCartButton: { css: '#add_to_cart' },
+  cartCheckout: ('Proceed to checkout'),
+
   async getProductPrice() {
     return await I.grabTextFrom(this.productPrice);
+    
+  },
+
+  clickAddToCart() {
+    I.click(this.addToCartButton);
+    I.wait(3)
```

Comment:
remove static wait. You can add dynamic wait to function, which is used after this one.